### PR TITLE
Test PR after line ending normalization

### PR DESCRIPTION
**Pull request by seanbudd**
This PR is a test sample for #12387 
This PR simulates an open PR to master, of a branch created before normalization, after normalization has occurred on master.

In this case `test-normalized-line-endings` has had all line endings changed to LF using the process in #12816 - this simulates origin/master after normalization.
`test-branch-from-master` has been branched from master, without normalization, and has had a small change applied to a file with CRLF line endings.

This shows that line ending normalization will cause most current PRs to have merge conflicts.

As a strategy to handle this, branches should merge master in the following way.
This example uses these test branches, so that devs can confirm the process locally.

1. Set the [merge strategy to renormalize for the repository](https://git-scm.com/docs/merge-config#Documentation/merge-config.txt-mergerenormalize):
    ```sh
    git config --local merge.renormalize true
    ```
1. Existing merge conflicts with master will still need to be resolved. See #PRTBA
    ```sh
    git merge origin/test-normalize-line-endings # To be replaced with origin/master
    ```
    
In summary, the final script will look something like
```sh
git config --local merge.renormalize true
For each pr:
  gh pr checkout pr.num
  git fetch origin/master
  git merge origin/master
  # if the merge is successful, push, else ?
```

### Message for contributors

```md
NVAccess has updated the NVDA repository to make the line endings normalized to LF. 
This created merge conflicts, which we have resolved using the methodology discussed in #12864.
For more information see the following:
* Project overview: #12387
* Detailed implementation and justification: #12816.
```

### CODE REVIEW

**Comment by feerrenrut** [comment]
I think we'll have to update all the PR's ourselves. Additionally, can you prepare an explainer comment that can be posted onto each PR?

It might be possible to script this with `gh` CLI tool.

**Comment by seanbudd** [comment]
@feerrenrut - What are your thoughts on the following?

NVAccess has updated the nvda repo to make the line endings consistent and normalized. 
This created merge conflicts, which we have resolved using the methodology discussed in #12864.
For more information see the following:
* Project overview: #12387
* Detailed implementation and justification: #12816.

This might help to make suggestions:

```diff
- NVAccess has updated the nvda repo to make the line endings consistent and normalized. 
- This created merge conflicts, which we have resolved using the methodology discussed in #12864.
- For more information see the following:
- * Project overview: #12387
- * Detailed implementation and justification: #12816.
+ NVAccess has updated the nvda repo to make the line endings consistent and normalized. 
+ This created merge conflicts, which we have resolved using the methodology discussed in #12864.
+ For more information see the following:
+ * Project overview: #12387
+ * Detailed implementation and justification: #12816.
```

I'll look into a script to perform this for all PRs.

**Comment by seanbudd** [comment]
I have a major concern - how do we handle PRs that have merge conflicts currently?
I think instead the script will have to be something like this, and we will have to handle the merge conflicts manually.

```sh
git checkout origin/master .gitattributes
git add --renormalize .
git commit -m "normalize line endings"
```

**Comment by feerrenrut** [comment]
When a PR has merge conflicts, either us or the author is expected to resolve. There probably needs to be a manual step to confirm that the merge went correctly.

**Comment by seanbudd** [comment]
I don't think our script would work in this case, and we will not be able to merge in master to resolve conflicts created from the normalization.

**Comment by seanbudd** [comment]
If we set [`git config merge.renormalize true`](https://git-scm.com/docs/merge-config#Documentation/merge-config.txt-mergerenormalize) we can perform these merges without conflicts directly.
